core/test/test_container.cpp

```diff
@@ -685,3 +686,52 @@ TEST(Fallback, failing) {
 	EXPECT_FALSE(t.plan());
 	EXPECT_EQ(t.solutions().size(), 0u);
 }
+
+TEST(Fallback, ConnectStageInsideFallbacks) {
```

Comment:
This fails for the following reason:
1. The `Generator` spawns a solution, which is correctly forwarded to the start pull interface of `Connect`.
2. However, because there was no state pushed into the end pull interface of `Connect`, `canCompute()` returns false
and the `Fallbacks` container forwards to the next child (which doesn't exist).

This can only be solved, if we drop the simple linear execution order and switch to topological execution order.

Reply:
Sounds reasonable to disable this test for now.